package.json

```diff
@@ -33,12 +37,15 @@
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
     "babel-jest": "^23.6.0",
+    "cypress": "^3.4.1",
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^6.1.0",
     "eslint-plugin-prettier": "^3.1.0",
     "eslint-plugin-vue": "^5.0.0",
     "husky": "^2.7.0",
+    "patch-package": "^6.1.2",
```

Comment:
Is this still needed? Did it help with anything?

Reply:
Yes, it did. I have used this package to patch and that effectively removed the pre-commit error that I had. So, this package is needed until cypress updates their repo with this fix.